src/EditModel/EditDataEntryForm/EditDataEntryForm.component.js

```diff
@@ -0,0 +1,324 @@
+import React, { useState, useRef, useEffect } from 'react';
```

Comment:
See https://github.com/dhis2/maintenance-app/blob/224f2ec40242776d1e92ab5d6fdc2cc9d4989c88/src/EditModel/EditDataEntryForm.component.js for previous class-based version of this component.